ch5/tutorial/crud_pydantic.py

```diff
@@ -0,0 +1,54 @@
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+
+
+class CreateIn(BaseModel):
+    name: str
+    nickname: str
+
+
+class CreateOut(BaseModel):
+    status: str
+    id: int
+
+
+# Create a FastAPI instance
+app = FastAPI()
+
+# User database
+USER_DB = {}
+
+# Fail response
+NAME_NOT_FOUND = HTTPException(status_code=400, detail="Name not found.")
+
+
+@app.post("/users", response_model=CreateOut)
+def create_user(user: CreateIn):
```

Comment:
```suggestion
def create_user(user: CreateIn) -> CreateOut:
```
output의 type도 명확하게 적어주면 좋을 것 같습니다. `respense_model` 과도 일치하는게 보이도록 하면 좋을 것 같아요.

Reply:
반영하였습니다  :)